src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/IContentHandler.cs

```diff
@@ -13,7 +14,9 @@ public interface IContentHandler
         Task LoadingAsync(LoadContentContext context);
         Task LoadedAsync(LoadContentContext context);
         Task ImportingAsync(ImportContentContext context);
+        Task BeforeImportAsync(IEnumerable<ImportContentContext> contentItems);
         Task ImportedAsync(ImportContentContext context);
+        Task AfterImportAsync(IEnumerable<ImportContentContext> contentItems);
```

Comment:
I feel these methods doesn't belong here in same interface -
 May be - 
- defining  `IBulkContentHandler` and/or it could inherits from this `IContentHandler` 
it could have BulkImprting and BulkImported method with  `BulkImportContentContext` having list of content items as propery

Reply:
At the moment AfterImportAsync is barely implemented; we just need to change its input parameter in DefaultContentManager.
Then just change some of the other implementations

Consider the following implementations

```
OrchardCore.Contents.Workflows.Handlers.ContentsHandler
OrchardCore.Lists.Drivers.LocalizationContainedPartHandler
OrchardCore.Contents.AuditTrail.Handlers.AuditTrailContentHandler
OrchardCore.Search.Lucene.Handlers.LuceneIndexingContentHandler
OrchardCore.Indexing.CreateIndexingTaskContentHandler
OrchardCore.Search.Elasticsearch.Core.Handlers.ElasticIndexingContentHandler
```

Reply:
Yeah, I'd add a separate event handler for this too. `IContentHandler` is all about singular content items. I second `IBulkContentHandler`.

Reply:
Ok, I see what you mean. I will finish this PR sometime

Reply:
Thank you!